File: sorting/selection_sorting/sorting.py
def selection_sort(arr):
    n = len(arr)

    for i in range(n - 1):
        min_index = i

        # Find the index of the minimum element in the remaining unsorted array
        for j in range(i + 1, n):
            if arr[j] < arr[min_index]:
                min_index = j

        # Swap the found minimum element with the first element of the unsorted array
        if min_index != i:
            arr[i], arr[min_index] = arr[min_index], arr[i]

    return arr

# Example usage:
arr = [64, 25, 12, 22, 11]
print("Original array:", arr)
sorted_arr = selection_sort(arr)
print("Sorted array:", sorted_arr)

File: sorting/selection_sorting/sorting.js
function selection_sort(arr) {
    let n = arr.length
    for (let i = 0 ; i < n-1 ; i++){
        let min_index = 0
        for (let j=i+1; j<n ; j++){
            if (arr[j] < arr[min_index]){
                min_index = j
            }
        }

        if (min_index!==i){
            let temp = arr[i];
            arr[i] = arr[min_index];
            arr[min_index] = temp
        }
    }
    return arr
}

const arr = [64, 25, 12, 22, 11];
console.log("Original array:", arr);
const sortedArr = selection_sort(arr);
console.log("Sorted array:", sortedArr);